examples/table.tape

```diff
@@ -1,19 +1,30 @@
 # This is a vhs script. See https://github.com/charmbracelet/vhs for more info.
 # To run this script, install vhs and run `vhs ./examples/table.tape`
 Output "target/table.gif"
-Set Theme "Aardvark Blue"
-Set Width 1200
-Set Height 800
+Set Width 1600
+Set Height 864
 Hide
 Type "cargo run --example=table --features=crossterm"
 Enter
 Sleep 1s
 Show
 Sleep 2s
-Set TypingSpeed 0.5s
+Set TypingSpeed 0.75s
 Down 2
 Up 2
-Down 8
-Up 12
-Down 4
+Down 2
+Right 1
+Down 2
+Right 1
```

Comment:
These color changes make this subjectively still a bit frantic for my liking.

It's difficult to get a good balance for something that you can look at and get an understanding of how the example is made up. In most of the examples, you'll see lots of ~2 second sleeps for that reason any time the view changes (this color change counts), or at the end of movement / typing / scrolling to allow the user to process things.

The opposite side of this is going too slow so the UI looks clunky.

Reply:
I'll play with table.tape to see what could be done

Reply:
I think It's better now what do you think?

Reply:
Looks good.